src/code/components/challenge-award.js

```diff
@@ -10,7 +10,7 @@ class ChallengeAwardView extends React.Component {
   };
 
   static defaultProps = {
-     challengeAwards: {"caseId":0, "challengeId":0, "challengeCount":0, "progress":[]},
+     challengeAwards: {"missionId":0, "challengeId":0, "challengeCount":0, "progress":[]},
```

Comment:
This is the only part I'm not sure about- I renamed `caseId` to `missionId`, but I didn't add `levelId` anywhere in this class. The awards seem to work just fine, even though I might expect there to be an issue with duplicate IDs without the level number.